pymatgen/analysis/ferroelectricity/ZVAL.yaml

```diff
@@ -0,0 +1,305 @@
+VASP_PBE_ZVAL:
```

Comment:
I've removed ZVAL and associated functions and added the additions to Outcar mentioned above. Now, if LCALCPOL = True, Outcar will parse the property zval_dict. I've added a test for this property.